Lombiq.Hosting.Tenants.Maintenance/StaggeredMigrations.cs

```diff
@@ -0,0 +1,85 @@
+using Lombiq.Hosting.Tenants.Maintenance.Constants;
+using Lombiq.Hosting.Tenants.Maintenance.Models;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentFields.Settings;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Data.Migration;
+using System.Threading.Tasks;
+
+namespace Lombiq.Hosting.Tenants.Maintenance;
+
+public sealed class StaggeredMigrations : DataMigration
+{
+    private readonly IContentDefinitionManager _contentDefinitionManager;
+
+    public StaggeredMigrations(IContentDefinitionManager contentDefinitionManager) =>
+        _contentDefinitionManager = contentDefinitionManager;
+
+    public async Task<int> CreateAsync()
+    {
+        await _contentDefinitionManager.AlterPartDefinitionAsync(nameof(StaggeredMaintenancePart), part => part
+            .WithField<NumericField>(nameof(StaggeredMaintenancePart.ProcessingStep), field => field
+                .WithDisplayName("Processing Step")
+                .WithSettings(new NumericFieldSettings
+                {
+                    Hint = "The number of tenants to be processed in each step or in case of parallel processing the" +
+                           " number of tenants started at once.",
+                }))
+            .WithField<TimeField>(nameof(StaggeredMaintenancePart.TimeSpanBetweenBatches), field => field
+                .WithDisplayName("Time Span Between Batches")
+                .WithSettings(new TimeFieldSettings
```

Comment:
It should just be a simple counter to define the number of seconds (also see my other comment amount not needing millisecond-precision). BTW for millisecond-precision this editor wouldn't be suitable anyway.

Reply:
You can't define milliseconds, because the step in the TimeFieldSettings is set to "1".

Reply:
I know, that's why I'm saying that it's not suitable for milliseconds. Previously this configuration was stored as `StaggeredMaintenanceOptions.MillisecondBetweenBatches` until I suggested to rename it to `BatchIntervalSeconds` (and change the logic accordingly).

Reply:
This is still a TimeField, but IMO it's confusing UI-wise - this is how it looks like when set to 20 seconds:
![image](https://github.com/user-attachments/assets/96d8a5af-6e3b-4c77-a62c-ba7b11706469)
I still think that a simple (unsigned) integer value, i.e. NumericField (BatchIntervalSeconds) with a value at least 0 would be a lot more straightforward.

Reply:
This is weird, because I had it working like 00:00:20 for seconds, but if this isn't working like you showed, then it is not worth fighting for this from my side 😄 So I changed it now to a NumericField.

Reply:
It's browser-dependent too (the screenshot is from Brave), but for simplicity's sake IMO seconds is enough. The most that could make sense to configure here is couple minutes (~ few hundred seconds), but in reality (for us, at least) it will be definitely be less than 60 seconds.